.travis.yml

```diff
@@ -1,11 +0,0 @@
-language: node_js
-node_js:
-  - "10"
-  - "12"
-  - "14"
-  - "15"
-before_install:
-  - npm --global install npm
```

Comment:
This step should be preserved, so that we use the latest version of npm in CI.

Reply:
Sure, done!

Reply:
Actually, as said, `npm@7` fails with invalid `peerDependencies`, so that's why the CI is failing currently, but once #531 has been merged, everything will work fine! :smile: